### DISCUSSION TITLE
Gradio Api

### DISCUSSION THREAD

**Discussion by UnbreakableBhanot** [lllyasviel/Fooocus#3178] [Q&A] [answered]
Hey i am confused with Gradio API , i just need to give image from my pc and a prompt to local host running foocus.
and I have to convert image to either bit64 or a link to do this
+ i like want everything cli based
like i run script then i get image in same location but not working in my case

**Comment by UnbreakableBhanot** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/157055209/d56c8f82-a2e4-4a55-b290-3920696af1aa)

check this image no output

**Reply by mashb1t** [reply, marked as answer]
you can find a working example in https://github.com/toutjavascript/FoooXus-Fooocus-Extender/blob/main/src/api.py and compare the parameters with your code.
